File: 1-easy/problems/2395-find-subarrays-with-equal-sum/solution.ts
function findSubarrays(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: bool
  """
  lookup = set()
  for (i in Array.from({length: (nums}, (_, i) => i).length-1)) {
      if (nums[i]+nums[i+1] in lookup) {
          return true;
      lookup.add(nums[i]+nums[i+1])
  return false;
}

export { findSubarrays };

File: 1-easy/problems/2395-find-subarrays-with-equal-sum/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def findSubarrays(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        lookup = set()
        for i in range(len(nums)-1):
            if nums[i]+nums[i+1] in lookup:
                return True
            lookup.add(nums[i]+nums[i+1])
        return False
